src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/custom.py

```diff
@@ -111,6 +109,7 @@ def update_application(client, resource_group_name, account_name, application_id
 
 def _upload_package_blob(package_file, url):
     """Upload the location file to storage url provided by autostorage"""
+    BlockBlobService = get_sdk_attr('azure.multiapi.storage.blob#BlockBlobService')
```

Comment:
`azure.multiapi.storage.blob` doesn't actually exist.
`azure.multiapi.storage.vYYYY_MM_DD.blob` is what exists and the helper method extracts that.